Jan2016/CJ/com/two95/aop/Authenticate.java

```diff
@@ -0,0 +1,25 @@
+package com.two95.aop;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+public class Authenticate {
+	public static void check()throws Exception
+	{
+		ApplicationContext ctx=new ClassPathXmlApplicationContext("bankingConf.xml");
+		Customer c=(Customer) ctx.getBean("cust");
+		String user1=c.getUsername();
+		
+		String pass=c.getPassword();
+		
+        if(user1.equals("ch")&&pass.equals("cj")){  
+        	
+            BankingOperation.withdrawal();  

```

Comment:
You should not bind code here. This authentication should be done by AOP.
When you call withdraw method,check() method should be executed